src/test/java/org/commonwl/view/cwl/CWLServiceTest.java

```diff
@@ -134,19 +138,24 @@ public void parseLobSTRv1WorkflowNative() throws Exception {
         testLobSTRWorkflow(lobSTRv1, true);
     }
     
-    /**
-     * Test native loading parsing of optional inline types
-     */
-    @Test
-    public void parseWorkflowInlineOptionalTypesNative() throws Exception {
-        CWLService cwlService = new CWLService(rdfService, new CWLTool(), 5242880);
-        Workflow wkflow = cwlService.parseWorkflowNative(
-                Paths.get("src/test/resources/cwl/oneline_optional_types.cwl"), null);
-        assertEquals(wkflow.getInputs().get("qualified_phred_quality").getType(), "int?");
-        assertEquals(wkflow.getInputs().get("ncrna_tab_file").getType(), "File?");
-        assertEquals(wkflow.getInputs().get("reverse_reads").getType(), "File?");
-        
-    }
+	/**
+	 * Test native loading parsing of optional inline types
+	 */
+	@Test
+	public void parseWorkflowInlineOptionalTypesNative() throws Exception {
+		CWLService cwlService = new CWLService(rdfService, new CWLTool(), 5242880);
+//        URI uri = new URI("https://github.com/emo-bon/pipeline-v5/raw/develop/workflows/gos_wf.cwl");
+//        InputStream is = uri.toURL().openStream();
+//        Workflow wkflow = cwlService.parseWorkflowNative(is, null, uri.toURL().getFile());
+		Workflow wkflow = cwlService.parseWorkflowNative(Paths.get("src/test/resources/cwl/oneline_optional_types.cwl"),
+				null);
+		assertEquals(wkflow.getInputs().get("qualified_phred_quality").getType(), "int?");
+		assertEquals(wkflow.getInputs().get("ncrna_tab_file").getType(), "File?");
+		assertEquals(wkflow.getInputs().get("reverse_reads").getType(), "File?");
+		assertEquals(wkflow.getInputs().get("ssu_tax").getType(), "string, File");
+		assertEquals(wkflow.getInputs().get("rfam_models").getType(), "{type=array, items=[string, File]}");
```

Comment:
I think other parts of the file use spaces instead of tabs? (didn't check the whole file, just the parts appearing here in GH UI, above and below your change).

Reply:
Ah, I read this test before reading the CWLService change. That file appears to be using tabs. No need to change anything, the project appears to have a mix of both.

Reply:
I used the source format option in Eclipse. If you recommend a different formatter then I'll apply that to the entire codebase in a separate PR 👍